tests/Auth/AuthorizeMiddlewareTest.php

```diff
@@ -318,14 +316,8 @@ public function testModelInstanceAsParameter()
 
         $request = m::mock(Request::class);
 
-        $nextParam = null;
-
-        $next = function ($param) use (&$nextParam) {
```

Comment:
$nextParam and $next are useless because they do nothing and don't have any assertions. This test is to ensure if right parameters are passed to Authorize::handle() methods.